batch/hail-ci-build.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash
 set -ex
 
-. ../loadconda
 conda activate hail-batch
 
-flake8 batch
-pylint batch --rcfile batch/pylintrc --score=n
```

Comment:
These seem to be duplicates because test-local-in-cluster depends on $(PY_CHECKERS)?

Reply:
👍